stories/Page.stories.tsx

```diff
@@ -70,11 +70,14 @@ Default.args = {
   errorCode: null,
```

Comment:
Nit: I think we should go ahead and pull out the `errorCode` control. It doesn't make as much sense anymore if we have a separate story for Errored and because we are no longer displaying the specific error code to the user.

Reply:
since errorCode is an actual HomeProp, this line just makes the type show up properly (a number instead of a JSON) rather than whether the arg shows up or not.